langchain/src/llms/base.ts

```diff
@@ -88,10 +96,15 @@ export abstract class BaseLLM extends BaseLanguageModel {
       this.callbacks,
       { verbose: this.verbose }
     );
+    const invocationParams = { invocation_params: this?.invocationParams() };
     const runManager = await callbackManager_?.handleLLMStart(
       { name: this._llmType() },
-      prompts
+      prompts,
+      uuidv4(),
```

Comment:
This should be undefined

Reply:
Makes sense! TY! I'm assuming that means explicitly passing `undefined`